<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(192, 225, 236);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(1, 35, 60);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../QRC/imagenesInicioSesion.qrc">
              <normaloff>:/newPrefix/Icons/Exterior.png</normaloff>:/newPrefix/Icons/Exterior.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../QRC/imagenesInicioSesion.qrc">
              <normaloff>:/newPrefix/Icons/User_Groups_Icon.png</normaloff>:/newPrefix/Icons/User_Groups_Icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../QRC/imagenesInicioSesion.qrc">
              <normaloff>:/newPrefix/Icons/Chat_Room_Icon.png</normaloff>:/newPrefix/Icons/Chat_Room_Icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../QRC/imagenesInicioSesion.qrc">
              <normaloff>:/newPrefix/Icons/Events_Icon.png</normaloff>:/newPrefix/Icons/Events_Icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../QRC/imagenesInicioSesion.qrc">
              <normaloff>:/newPrefix/Icons/Adjust_Icon.png</normaloff>:/newPrefix/Icons/Adjust_Icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(1, 35, 60);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,5,1">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_7">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,6,1,6,1">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="frame_9">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_chat">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(0, 75, 115);

border-radius: 20px
</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5" stretch="2,8">
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="topMargin">
                    <number>10</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>40</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame_10">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QPushButton" name="pushButton_6">
                        <property name="minimumSize">
                         <size>
                          <width>80</width>
                          <height>80</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset resource="../QRC/imagenesInicioSesion.qrc">
                          <normaloff>:/newPrefix/Icons/Chat_Room_Icon.png</normaloff>:/newPrefix/Icons/Chat_Room_Icon.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>80</width>
                          <height>80</height>
                         </size>
                        </property>
                        <property name="flat">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QFrame" name="frame_15">
                        <property name="styleSheet">
                         <string notr="true"/>
                        </property>
                        <property name="frameShape">
                         <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_14">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(192, 225, 236);

border-radius: 0px</string>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_6">
                      <property name="leftMargin">
                       <number>20</number>
                      </property>
                      <property name="topMargin">
                       <number>20</number>
                      </property>
                      <property name="rightMargin">
                       <number>20</number>
                      </property>
                      <property name="bottomMargin">
                       <number>20</number>
                      </property>
                      <item>
                       <widget class="QScrollArea" name="scrollArea">
                        <property name="widgetResizable">
                         <bool>true</bool>
                        </property>
                        <widget class="QWidget" name="scrollAreaWidgetContents">
                         <property name="geometry">
                          <rect>
                           <x>0</x>
                           <y>0</y>
                           <width>294</width>
                           <height>274</height>
                          </rect>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">background-color: rgb(121, 184, 211);</string>
                         </property>
                        </widget>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_11">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_grupos">
                  <property name="styleSheet">
                   <string notr="true">background-color: rgb(0, 75, 115);






</string>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7" stretch="4,5,20">
                   <property name="leftMargin">
                    <number>30</number>
                   </property>
                   <property name="rightMargin">
                    <number>30</number>
                   </property>
                   <property name="bottomMargin">
                    <number>40</number>
                   </property>
                   <item>
                    <widget class="QFrame" name="frame_16">
                     <property name="styleSheet">
                      <string notr="true"/>
                     </property>
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,10">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                      <item>
                       <widget class="QPushButton" name="pushButton_7">
                        <property name="minimumSize">
                         <size>
                          <width>50</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="text">
                         <string/>
                        </property>
                        <property name="icon">
                         <iconset resource="../QRC/imagenesInicioSesion.qrc">
                          <normaloff>:/newPrefix/Icons/Conferencia.png</normaloff>:/newPrefix/Icons/Conferencia.png</iconset>
                        </property>
                        <property name="iconSize">
                         <size>
                          <width>50</width>
                          <height>50</height>
                         </size>
                        </property>
                        <property name="flat">
                         <bool>true</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <widget class="QFrame" name="frame_12">
                        <property name="styleSheet">
                         <string notr="true"/>
                        </property>
                        <property name="frameShape">
                         <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                         <enum>QFrame::Raised</enum>
                        </property>
                       </widget>
                      </item>
                     </layout>
                    </widget>
                   </item>
                   <item>
                    <widget class="QScrollArea" name="scrollArea_2">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(192, 225, 236);

</string>
                     </property>
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <widget class="QWidget" name="scrollAreaWidgetContents_2">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>313</width>
                        <height>73</height>
                       </rect>
                      </property>
                      <property name="styleSheet">
                       <string notr="true">.</string>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_9">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QListWidget" name="lista_grupos">
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <property name="verticalScrollBarPolicy">
                          <enum>Qt::ScrollBarAlwaysOff</enum>
                         </property>
                         <property name="horizontalScrollBarPolicy">
                          <enum>Qt::ScrollBarAlwaysOff</enum>
                         </property>
                         <property name="itemAlignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                   <item>
                    <widget class="QScrollArea" name="scrollArea_3">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(192, 225, 236);

</string>
                     </property>
                     <property name="widgetResizable">
                      <bool>true</bool>
                     </property>
                     <widget class="QWidget" name="scrollAreaWidgetContents_3">
                      <property name="geometry">
                       <rect>
                        <x>0</x>
                        <y>0</y>
                        <width>330</width>
                        <height>261</height>
                       </rect>
                      </property>
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <property name="leftMargin">
                        <number>0</number>
                       </property>
                       <property name="topMargin">
                        <number>0</number>
                       </property>
                       <property name="rightMargin">
                        <number>0</number>
                       </property>
                       <property name="bottomMargin">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QListWidget" name="lista_miembros"/>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="frame_13">
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_8">
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../QRC/imagenesInicioSesion.qrc"/>
 </resources>
 <connections/>
</ui>
